src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,59 @@
+import { FC } from 'react';
+import { FilterOptions } from '../../types/FilterOptions';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  currentFilter: FilterOptions;
+  onFilterOptionChange: (status: FilterOptions) => void;
+  onCompletedDelete: () => void;
+}
+
+export const Footer: FC<Props> = ({
+  todos,
+  currentFilter,
+  onFilterOptionChange,
+  onCompletedDelete,
+}) => {
+  const todosLeft = todos.reduce(
+    (current, todo) => current + (!todo.completed ? 1 : 0),
+    0,
+  );
+
+  const hasCompletedTodo = !todos.some(todo => todo.completed)
```

Comment:
The name hasCompletedTodo suggests you're checking for at least one completed todo, but the logic is checking if there are no completed todos.
This might confuse developers who read the code later.
```suggestion
  const hasNoCompletedTodo = !todos.some(todo => todo.completed)
```